vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -215,6 +216,56 @@ YGMeasureFunc ABIViewManager::GetYogaCustomMeasureFunc() const {
   }
 }
 
+void ABIViewManager::OnPointerPressed(
+    const ::Microsoft::ReactNative::ShadowNodeBase *node,
+    const ReactPointerEventArgs &args) {
+  if (m_viewManagerWithPointerEvents) {
+    m_viewManagerWithPointerEvents.OnPointerPressed(node->GetView(), args);
+  } else {
+    Super::OnPointerCaptureLost(node, args);
```

Comment:
OnPointerPressed (fix below as well).

Reply:
We should also call the base so 3rd party view managers do not need to implement custom behaviors for "box-none" / "box-only"